### PR TITLE
settings: update lint paths to be more comprehensive (bug 1918236)

### PR DESCRIPTION
**Pull request by zzzeid**
- update LINT_PATHS
- run `lando format`

### CODE REVIEW

File: src/lando/settings.py [hunk 1]
```diff
@@ -198,7 +196,10 @@
     "lando.main.auth.LandoOIDCAuthenticationBackend",
 ]
 
-LINT_PATHS = tuple(f"{BASE_DIR}/{path}" for path in ("api", "dockerflow", "main", "utils", "tests"))
+LINT_PATHS = tuple(
+    f"{BASE_DIR}/{path}"
+    for path in (".", "ui", "api", "dockerflow", "main", "utils", "tests")
+)
```
Comment (cgsheeh) [inline src/lando/settings.py:202]: Any reason we couldn't just run `black` on all files by default? Maybe it would make more sense to maintain a list of exclude paths in the `tool.black` section of `pyproject.toml` instead of maintaining a list of include paths?

File: src/lando/ui/legacy/api.py [hunk 2]
```diff
@@ -40,15 +39,16 @@ def post(self, request):
...
+            )
 
         if not uplift_request_form.validate():
             errors = list(chain(*uplift_request_form.errors.values()))
             return JsonResponse({"errors": errors}, status_code=400)
 
-        revision_id = uplift_request_form.revision_id.data
-        repository = uplift_request_form.repository.data
+        # revision_id = uplift_request_form.revision_id.data
+        # repository = uplift_request_form.repository.data
```
Comment (cgsheeh) [inline src/lando/ui/legacy/api.py:51]: Unrelated?

**Review by cgsheeh — CHANGES_REQUESTED** [review]
r- because of the commented uplift lines, otherwise this is good to go.

File: src/lando/ui/legacy/api.py (hunk 2 above)
Comment (zzzeid) [reply src/lando/ui/legacy/api.py:51]: They are unused variables that popped up in the lint check. There's additional commented out code below that is connected to these variables as well.

File: src/lando/settings.py (hunk 1 above)
Comment (zzzeid) [reply src/lando/settings.py:202]: These changes affect both `black` and `ruff` by the way.

I guess this is a remnant of the legacy implementation (we've always explicitly set the paths this way) though this is less relevant (or completely irrelevant perhaps) now since all relevant files are under `src/lando`.

In the future, we might want to add additional paths to this that are outside `src/lando`, so maybe this should just be changed to 
```suggestion
LINT_PATHS = (f"{BASE_DIR}/{path}", )
```

**Review by cgsheeh — APPROVED** [review]
